src/utils.rs

```diff
@@ -10,177 +23,154 @@ pub fn has_attribute(needle: &str, attributes: &[syn::Attribute]) -> bool {
     })
 }
 
+fn check_expression_is_path(expr: Expr) -> Option<ExprPath> {
```

Comment:
could I request you add a comment explaining this method?

Reply:
Sure, no problem. I took a shot in the latest commits. Please let me know what you think.